src/virtualize.js

```diff
@@ -111,7 +111,7 @@ export default function virtualize(MyComponent) {
       }
 
       if (onChangeIndex) {
-        onChangeIndex(index);
+        onChangeIndex(index, indexLatest);
```

Comment:
> I was expecting to receive the indexLatest as per the documentation in my virtualize HOC anyway.

> I can work around it by looking at this.state.index because that would hold the same info
 
`this.state.index` should be correct 👍 .